src/app/homepage/pages/recipes/sentry/sentry.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { BasePageComponent } from '../../page/page.component';
+
+@Component({
+  selector: 'app-sentry',
+  templateUrl: './sentry.component.html',
+  styleUrls: ['./sentry.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SentryComponent extends BasePageComponent {
+
+  get dependencies(): string {
+    return `
+$ npm install --save raven
+$ npm install --save-dev @types/raven`;
+  }
+
+  get expressExample(): string {
+    return `
+var app = require('express')();
+var Raven = require('raven');
+
+// Must configure Raven before doing anything else with it
+Raven.config('__DSN__').install();
+
+// The request handler must be the first middleware on the app
+app.use(Raven.requestHandler());
+
+app.get('/', function mainHandler(req, res) {
+    throw new Error('Broke!');
+});
+
+// The error handler must be before any other error middleware
+app.use(Raven.errorHandler());
+
+// Optional fallthrough error handler
+app.use(function onError(err, req, res, next) {
+    // The error id is attached to \`res.sentry\` to be returned
+    // and optionally displayed to the user for support.
+    res.statusCode = 500;
+    res.end(res.sentry + '\\n');
+});
+
+app.listen(3000);`;
+  }
+
+  get sentryInterceptor () {
+    return `
+import {ExecutionContext, NestInterceptor} from "@nestjs/common";
+import {Observable} from "rxjs/Observable";
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/throw';
+import * as Raven from 'raven';
+
+@Interceptor()
+class SentryInterceptor implements NestInterceptor {
+  intercept(dataOrRequest, context: ExecutionContext, stream$: Observable<any>): Observable<any> {
+    return stream$.catch(err => {
+      Raven.captureException(err);
+      return Observable.throw(err);
```

Comment:
I'd recommend you to use `_throw` here instead of relying on the monkey patching.

`import {_throw} from 'rxjs/observable/throw';`

And use catch with `.pipe(catchError())`

Reply:
I'm not used to `rxjs` what is the difference between `Observable.throw` and `_throw`?
Anyway, I'm getting an error when trying to use the latest:
```
Error: Cannot find module 'rxjs/add/operator/throw'
```

Reply:
Meh, don't mind the error I was getting, it was my mistake

Reply:
`Observable.throw` relies on monkey patching of the observable or on the full import of RxJS (way to large).

It is recommended to use pipe-able operators in order to make sure that the operator is included and in order to not create a giant bundle. Take a closer look at [this](https://github.com/ReactiveX/rxjs/blob/master/doc/pipeable-operators.md) for further information.